RecoEgamma/EgammaTools/src/EgammaPCAHelper.cc

```diff
@@ -1,13 +1,16 @@
+
 #include "RecoEgamma/EgammaTools/interface/EgammaPCAHelper.h"
 
 #include "DataFormats/HGCRecHit/interface/HGCRecHit.h"
 #include "DataFormats/Math/interface/deltaPhi.h"
 #include "DataFormats/ForwardDetId/interface/HGCalDetId.h"
 #include "DataFormats/HcalDetId/interface/HcalDetId.h"
 
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
 // To retrieve HGCalImagingAlgo::maxlayer
 // Also available as HGCal3DClustering::lastLayerBH and HGCalDepthPreClusterer::lastLayerBH
-#include "RecoLocalCalo/HGCalRecAlgos/interface/HGCalImagingAlgo.h"
+#include "RecoLocalCalo/HGCalRecProducers/interface/HGCalClusteringAlgoBase.h"
```

Comment:
is there a reason this algorithm header is located in the `HGCalRecProducers` package?

Reply:
Not sure I understand the question. Maybe you missed the fact that all algos have been moved here and modified to be plugins?

Reply:
Yes, I did miss this, thanks for pointing it out.